app/views/users/index.html.erb

```diff
@@ -1,3 +1,3 @@
-<h1>List of Users</h1>
-<p><%= @placeholder_text %></p>
-<p>Find me in app/views/users/index.html.erb</p>
+<section id="users">
+  <%= render partial: 'user', collection: @users %>
+</section>
```

Comment:
![image](https://github.com/richie1988/blog-app/assets/93039359/2c1ae1e3-414b-4c53-8568-d2578fc45cb3)
![image](https://github.com/richie1988/blog-app/assets/93039359/f195ebfd-b046-45d1-9f49-d1802426ea39)
![image](https://github.com/richie1988/blog-app/assets/93039359/cb1367a8-51ac-45a2-b06a-6a2e410902ad)
![image](https://github.com/richie1988/blog-app/assets/93039359/b57438b1-2187-4256-900c-af807d8b56d9)

- [ ] Great work so far but as the previous reviewer suggested it would be great if your content should expand on the whole screen to follow the wireframe :+1: .